docs/changelog.md

```diff
@@ -100,6 +100,8 @@ permalink: /changelog/
    * ⚠️ **This is a breaking API change**: Custom pings must be explicitly
      registered with Glean at startup time. See
      `components/service/glean/docs/pings/custom.md` for more information.
+   * Disabling telemetry through `setUploadEnabled` now clears all metrics
```

Comment:
Let's call out that this includes the 'client_id' but not the 'first_run_date'

Reply:
Done.  Also had to move this to a new section of the changelog...